### PR TITLE
Translate case list registration form label

### PR DESCRIPTION
**Pull request by kaapstorm**
Bulk translate does not offer the case list registration form label to be translated. This change resolves that. cf. [FB 189956](http://manage.dimagi.com/default.asp?189956) for more context.

Probably easier to review per commit. @orangejenny, `git blame` makes me think you're the best person to eye-ball this. Buddy @dannyroberts

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 1]
```diff
@@ -142,6 +143,7 @@ class BulkAppTranslationBasicTest(BulkAppTranslationTestBase):
           ("Form", "module1_form1", "My more & awesome form", "", "", "", "", "", "", "", "93ea2a40df57d8f33b472f5b2b023882281722d4")
         )),
         ("module1", (
+          ("case_list_form_label", "list", "Register Mother", "Inscrivez-Mère"),
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: E121 continuation line under-indented for hanging indent

File: corehq/apps/app_manager/translations.py [hunk 2]
```diff
@@ -258,6 +258,12 @@ def expected_bulk_app_sheet_rows(app):
         # Populate module sheet
         rows[module_string] = []
         if not isinstance(module, ReportModule):
+            if module.case_list_form.form_id:
+                # Add row for label of case list registration form
+                rows[module_string].append(
+                        ('case_list_form_label', 'list') +
```
Comment (dimagimon) [inline corehq/apps/app_manager/translations.py]: E126 continuation line over-indented for hanging indent

File: corehq/apps/app_manager/translations.py [hunk 3]
```diff
@@ -705,6 +716,24 @@ def update_case_list_translations(sheet, rows, app):
         return msgs
 
     # Update the translations
+    def _update_translation(row, language_dict, require_translation=True):
+        ok_to_delete_translations = (
+            not require_translation or has_at_least_one_translation(
+                    row, 'default', app.langs
```
Comment (dimagimon) [inline corehq/apps/app_manager/translations.py]: E126 continuation line over-indented for hanging indent

**Comment by orangejenny** [comment]
looks good to me

**Pull request merged**